autoscheduler/frontend/src/components/LandingPage/SelectTerm/SelectTerm.tsx

```diff
@@ -14,19 +16,43 @@ const options = [
   'Semester 6',
 ];
 
+// Maps between the term description and its actual value (which we'll use to store)
+// We might not need this in the future?
```

Comment:
Is this termMap going to come from the bcakend in the future? Even so, for now, I would set it to something like "Fall 19", "Fall 19(1)", and "Fall 19(2)" instead of "Semester 1" so that we know what test data we're looking at

Reply:
Yeah it'll come from #147 . We might need to change the data structure of it to whatever api/terms uses but we can deal with it in that issue as it won't be that much to change regardless